.env.sample

```diff
@@ -1,3 +1,6 @@
 TESTNET_ACCOUNT_PRIVATE_KEY=private key for an account we have on testnet that has eth.
 TESTNET_CHAIN_URL=url to connect to testnet
 LOCAL_NETWORK_ACCOUNT_PRIVATE_KEY=private key for an account we have on our local network that has eth.
+VALIDATOR1=address for validator running on node called liberty-chain
+VALIDATOR2=address for validator running on node called liberty-chain1
+BOOTNODE=enode address of node this node would like to sync with
```

Comment:
Are these 0x prefixed addresses or does it not matter?

Reply:
it does not matter but I can specify.